### PR TITLE
doc: improve the API documentation of VideoStreamSender

### PR DESCRIPTION
**Pull request by toru-unity**

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/UnityRenderStreaming?pullRequest=1016) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/UnityRenderStreaming?pullRequest=1016) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/UnityRenderStreaming?pullRequest=1016) it.</sub>

File: com.unity.renderstreaming/Runtime/Scripts/VideoStreamSender.cs [hunk 1]
```diff
@@ -410,9 +437,15 @@ public void SetBitrate(uint minBitrate, uint maxBitrate)
         }
 
         /// <summary>
-        ///
+        /// Sets The factor of multiply with the video resolution to reduce the bandwidth.
```
Comment (chrstphrsxtn) [inline com.unity.renderstreaming/Runtime/Scripts/VideoStreamSender.cs:440]:
```suggestion
        /// Sets the scale factor by which to reduce the video resolution to conserve bandwidth.
```

File: com.unity.renderstreaming/Runtime/Scripts/VideoStreamSender.cs [hunk 2]
```diff
@@ -410,9 +437,15 @@ public void SetBitrate(uint minBitrate, uint maxBitrate)
...
         /// </summary>
-        /// <param name="scaleFactor">The parameter must be greater than 1.0f.</param>
+        /// <example>
+        /// <code>
+        /// videoStreamSender.SetScaleResolutionDown(2.0f);
+        /// </code>
+        /// </example>
+        /// <param name="scaleFactor">The scale factor to reduce the resolution. Must be greater than 1.0f.</param>
```
Comment (chrstphrsxtn) [inline com.unity.renderstreaming/Runtime/Scripts/VideoStreamSender.cs:447]:
```suggestion
        /// <param name="scaleFactor">The scale factor by which to reduce the resolution. Must be greater than 1.0f.</param>
```

File: com.unity.renderstreaming/Runtime/Scripts/VideoStreamSender.cs [hunk 3]
```diff
@@ -267,39 +267,39 @@ public WebCamTexture sourceWebCamTexture
...
+        /// Gets the maximum bitrate of the video stream.
         /// </summary>
         public uint maxBitrate
         {
             get { return m_Bitrate.max; }
         }
 
         /// <summary>
-        ///
+        /// The factor of multiply with the video resolution to reduce the bandwidth.
```
Comment (chrstphrsxtn) [inline com.unity.renderstreaming/Runtime/Scripts/VideoStreamSender.cs:294]:
```suggestion
        /// The scale factor by which to reduce the video resolution to conserve bandwidth.
```

**Review by chrstphrsxtn — APPROVED** [review]
Approving so as not to block but with some minor suggestions to adjust the wording of "factor of multiply" which isn't quite standard English.

**Pull request merged**